cloudformation/template.yaml

```diff
@@ -100,6 +106,11 @@ Parameters:
   #   Default: DevPortalMarketplaceSubscriptionTopic
 
   CognitoIdentityPoolName:
+    Type: String
+    Description: The name for your legacy identity pool. Set this only if you have an existing instance to fail over to and before updating it.
+    Default: ''
+
+  CognitoIdentityPoolName2:
```

Comment:
Why not CognitoIdentityPoolNameNew or *V2 or something else more descriptive?

Reply:
V2 could work, though I should've kept the number consistent with the scheme used in the pools.

Reply:
My main concern on the names is so that our users don't get confused. "Why do I need/should I have two user pools?"